utils/litemap/src/map.rs

```diff
@@ -51,41 +47,13 @@ impl<K, V, S> LiteMap<K, V, S> {
 }
 
 impl<K, V> LiteMap<K, V, Vec<(K, V)>> {
-    /// Convert a `Vec<(K, V)>` into a [`LiteMap`].
-    ///
-    /// # Safety
-    ///
-    /// The vec must be sorted and have no duplicate keys.
```

Comment:
Nit: Add back the

```
/// The slice must be sorted and have no duplicate keys.
```

It doesn't need to say `# Safety`, but we should retain the documentation.

Reply:
Done.